From f7362232f47a9fcaf0162087dbbbdb0b4562b59d Mon Sep 17 00:00:00 2001
From: Remi Collet <remi@php.net>
Date: Thu, 31 Jan 2013 14:22:56 +0100
Subject: [PATCH] Fix failed test: sys_errlist[116] have changed on recent
 glibc (Fedora 18) old: Stale NFS file handle new: Stale
 file handle

---
 ext/sockets/tests/socket_strerror.phpt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/sockets/tests/socket_strerror.phpt b/ext/sockets/tests/socket_strerror.phpt
index 7985fad..fb9925e 100644
--- a/ext/sockets/tests/socket_strerror.phpt
+++ b/ext/sockets/tests/socket_strerror.phpt
@@ -138,7 +138,7 @@ string(12) "Host is down"
 string(16) "No route to host"
 string(29) "Operation already in progress"
 string(25) "Operation now in progress"
-string(21) "Stale NFS file handle"
+string(%d) "Stale%sfile handle"
 string(24) "Structure needs cleaning"
 string(27) "Not a XENIX named type file"
 string(29) "No XENIX semaphores available"
-- 
1.7.11.5

From 9d75bf35e96bfc5c8d629ecef6807a90b4c98be7 Mon Sep 17 00:00:00 2001
From: Remi Collet <remi@php.net>
Date: Thu, 31 Jan 2013 14:38:39 +0100
Subject: [PATCH] Mark this test as requiring internet connecion.

---
 ext/sockets/tests/socket_bind.phpt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/ext/sockets/tests/socket_bind.phpt b/ext/sockets/tests/socket_bind.phpt
index 15181e6..7ea2df8 100644
--- a/ext/sockets/tests/socket_bind.phpt
+++ b/ext/sockets/tests/socket_bind.phpt
@@ -8,6 +8,9 @@ fa@php.net
     if (!extension_loaded('sockets')) {
         die('skip - sockets extension not available.');
     }
+    if (getenv("SKIP_ONLINE_TESTS")) {
+        die("skip test requiring internet connection");
+    }
 ?>
 --FILE--
 <?php
-- 
1.7.11.5

